Install.md

```diff
@@ -1,321 +1,224 @@
-    $3dTilesLayerDiscreteLOD.visible = true;
-
-    itowns.View.prototype.addLayer.call(view, $3dTilesLayerDiscreteLOD);
-
-    // Create a new Layer 3d-tiles For Viewer Request Volume
-    // -----------------------------------------------------
-    const $3dTilesLayerRequestVolume = new itowns.GeometryLayer('3d-tiles-request-volume', view.scene);
-
-    $3dTilesLayerRequestVolume.preUpdate = preUpdateGeo;
-    $3dTilesLayerRequestVolume.update = itowns.process3dTilesNode(
-        itowns.$3dTilesCulling,
-        itowns.$3dTilesSubdivisionControl
-    );
-
-    $3dTilesLayerRequestVolume.name = 'RequestVolume';
-    $3dTilesLayerRequestVolume.url = 'http://localhost:9090/getCity?city=citydb_v3';
-    $3dTilesLayerRequestVolume.protocol = '3d-tiles'
-    $3dTilesLayerRequestVolume.overrideMaterials = true;  // custom cesium shaders are not functional
-    $3dTilesLayerRequestVolume.type = 'geometry';
-    $3dTilesLayerRequestVolume.visible = true;
-
-    itowns.View.prototype.addLayer.call(view, $3dTilesLayerRequestVolume);
-
- ````  
-
-Run `npm start`
-Now open the resulting `http://localhost:8080/examples/planar_3dtiles.html` file with your favorite browser.
-
-*Note : if after running `npm start` you have an error message saying that node is not find in `/usr/bin/env`, try running the following : `sudo ln -s /usr/bin/nodejs /usr/bin/node`. Explainations : on ubuntu node.js package is installed in /usr/bin/nodejs and when npm starts, it runs a the command `/usr/bin/env node` which it doesn't find. `sudo ln -s /usr/bin/nodejs /usr/bin/node` allows to create a symbolic link between node and nodejs in order to allow npm to find nodejs.*
-
-Assert all is well by opening `http://localhost:8080/examples/planar.html` with your browser.
+    . venv/bin/activate
+    python tools/export_tileset -D tools/export_tileset_conf.yml -t
+````
+
+* run `3d-tiles-samples` server:
+
+Copy the tileset generated by py3dtiles into 3d-tiles-samples and run the server:
+````
+    cd <path/to/3d-tiles-samples>/tilesets
+    mkdir 4DTileset
+    cd <path/to/py3dtiles/>
+    cp -a tiles tileset.json <path/to/3d-tiles-samples>/tilesets/4DTileset
+    cd <path/to/3d-tiles-samples>
+    npm start
+````
+
+*Note: if you want your node server to be publicly accessible you need to run
+`./node_modules/.bin/nodemon --watch bin --watch lib --watch index.js server.js --public true` instead of `npm start`*
+
+Check that your tileset is accessible by opening
+http://localhost:8003/tilesets/4DTileset/tileset.json in your web browser.
+
+* run UDV:
+
+````
+    cd UDV
+````
+
+(Optionnal) If you want to use UDV with your local tileset, then edit UDV building-server query:
+
+open `UDV/UDV-Core/examples/Demo.js` with your favorite editor and change the
+line `buildingServerRequest` to http://localhost:8003/tilesets/4DTileset/tileset.json
+
+(end optionnal)
+
+````
+    cd UDV-Core
+    npm start
+````
+
+Open your web browser and go to `http://localhost:8080/examples/Demo.html`
```

Comment:
If not already done, make sure not content was lost (many lines were moved to other md files, but were all such  meaningful content displaced and pointed too ?). By the way it looks like InstallDeprecated.md serves this purpose, but if this file is for historical purposes then the git should be enough....

Reply:
I checked that nothing has been lost, hopefully I haven't missed anything. 

You're right about the InstallDeprecated.md, git keeps track of the changes and previous versions. However, I would like to keep it as is for couple of weeks (months ?) until our software architecture is stable enough and until we are sure we don't need an installation based on building-server component (or equivalent) anymore; if that's ok with you ?

Reply:
Oh well, I never read documents that are announced as deprecated anyhow :-)
So I'm at ease with that...